app/controllers/user_risk_controller.py

```diff
@@ -0,0 +1,61 @@
+from http import HTTPStatus
+
+from flask import request
+from flask_jwt_extended import decode_token, jwt_required
+from sqlalchemy.orm import Query, Session
+
+from app.configs.database import db
+from app.exceptions.user_exc import UserNotFound
+from app.models.risk_model import RiskModel
+from app.models.user_model import UserModel
+
+
+@jwt_required()
+def create_user_risk_profile():
```

Comment:
Please create a task to componentize the functions into services, there is no need to to that now, but later in the project a few of these functions might be useful to reuse.